SPECS-EXTENDED/dmraid/dmraid.spec

```diff
@@ -9,7 +9,7 @@ Distribution:   Mariner
 Summary: Device-mapper RAID tool and library
 Name: dmraid
 Version: 1.0.0.rc16
```

Comment:
Shouldn't there be a -3 here....instead of below where you added it on line 28.
Also, do we really still need a release candidate?

Reply:
I wasn't sure how to do release candidate since it was conflicting with our release number, but I found dmraid-1.0.0.rc16 anyway so this should be fine.